### PR TITLE
[NL] Improvements media_player intents

### PR DESCRIPTION
**Pull request by TheFes**
* Make all media_player intents area aware
* add area support to all media_player intents
* add additional options to `<media_item>` expansion rule

~~Other change~~
* ~~Add `wil je` to `skip_words`. This will allow sentences like `wil je het volgende liedje opzetten`. I've noticed my kids tend to include `wil je` when they try to ask something.~~ removed for now, was causing conflicts with other intents. Will add this in a later PR

### CODE REVIEW

File: sentences/nl/media_player_HassMediaUnpause.yaml [hunk 1]
```diff
@@ -3,8 +3,21 @@ intents:
...
+          - "hervat [[de |het ]<media_item> op ]<name>"
+          - " [[de |het ]<media_item> op ]<name> hervatten"
           - "[zet ]<name>[ weer][ op] ([af]spelen|play)"
+          - "ga verder met [de |het]<media_item> op <name>"
         requires_context:
           domain: media_player
+      - sentences:
+          - "[[de |het ]<media_item> ]hervat[ten]"
+          - "[zet ][[de |het ]<media_item> ][weer ][op ]([af]spelen|play)"
+          - "ga verder met [de |het]<media_item>"
```
Comment (frenck) [inline sentences/nl/media_player_HassMediaUnpause.yaml:15]:
```suggestion
          - "ga verder met [de |het ]<media_item>"
```

File: sentences/nl/media_player_HassMediaUnpause.yaml [hunk 2]
```diff
@@ -3,8 +3,21 @@ intents:
...
+        requires_context:
+          area:
+            slot: true
+      - sentences:
+          - "hervat [[de |het ]<media_item> <in> ]<area>"
+          - " [[de |het ]<media_item> <in> ]<area> hervatten"
+          - "[zet ]<area>[ weer][ op] ([af]spelen|play)"
+          - "ga verder met [de |het]<media_item> <in> <area>"
```
Comment (frenck) [inline sentences/nl/media_player_HassMediaUnpause.yaml:23]:
```suggestion
          - "ga verder met [de |het ]<media_item> <in> <area>"
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by TheFes** [comment]
Removed all spaces in optional parts.
Also added optional `<media_type>` mention to pause sentences (was already added to unmpause)

**Review by frenck — APPROVED** [review]
Thanks, @TheFes 👍 

../Frenck

**Pull request merged**